### PR TITLE
Test/MIDAZ-304

### PR DESCRIPTION
**Pull request by maxwelbm**
# Midaz Pull Request Checklist

## Pull Request Type
[//]: # (Check the appropriate box for the type of pull request.)

- [ ] Auth
- [ ] Infra
- [ ] Ledger
- [x] Mdz
- [ ] Transaction
- [ ] Pipeline
- [ ] Documentation

## Checklist
Please check each item after it's completed.

- [x] I have tested these changes locally.
- [x] I have updated the documentation accordingly.
- [x] I have added necessary comments to the code, especially in complex areas.
- [x] I have ensured that my changes adhere to the project's coding standards.
- [x] I have checked for any potential security issues.
- [x] I have ensured that all tests pass.
- [ ] I have updated the version appropriately (if applicable).
- [x] I have confirmed this code is ready for review.

## Additional Notes
[//]: # (Add any additional notes, context, or explanation that could be helpful for reviewers.)
## Obs: Please, always remember to target your PR to develop branch instead of main.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -151,6 +151,10 @@ all-services:
 	$(MAKE) -C $(LEDGER_DIR) $(COMMAND) && \
 	$(MAKE) -C $(TRANSACTION_DIR) $(COMMAND)
 
+<<<<<<< HEAD
+test_integration_cli:
+	go test -v -tags=integration ./components/mdz/test/integration/...
+=======
```
Comment (MartinezAvellan) [inline Makefile]: Pls, remove this merge comments. ❤️

File: Makefile [hunk 2]
```diff
@@ -162,4 +166,5 @@ tidy:
 generate-docs-all:
 	@echo "$(BLUE)Executing command to generate swagger...$(NC)"
 	$(MAKE) -C $(LEDGER_DIR) generate-docs && \
-	$(MAKE) -C $(TRANSACTION_DIR) generate-docs
\ No newline at end of file
+	$(MAKE) -C $(TRANSACTION_DIR) generate-docs
+>>>>>>> develop
```
Comment (MartinezAvellan) [inline Makefile]: Pls, remove this merge comments. ❤️

File: components/mdz/test/integration/testutils.go [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+	"regexp"
+	"testing"
+	"time"
+
+	"github.com/Netflix/go-expect"
+)
+
+const interactionDelay = 500 * time.Millisecond
+
+// up Send up arrow
```
Comment (MartinezAvellan) [inline components/mdz/test/integration/testutils.go:18]: Pls, remove this comments. ❤️

File: go.mod [hunk 4]
```diff
@@ -46,13 +46,20 @@ require (
...
 require (
 	github.com/KyleBanks/depth v1.2.1 // indirect
+	github.com/Netflix/go-expect v0.0.0-20220104043353-73e0943537d2 // indirect
 	github.com/atotto/clipboard v0.1.4 // indirect
 	github.com/aymanbagabas/go-osc52/v2 v2.0.1 // indirect
 	github.com/cenkalti/backoff/v4 v4.3.0 // indirect
 	github.com/cespare/xxhash/v2 v2.3.0 // indirect
 	github.com/charmbracelet/lipgloss v1.0.0 // indirect
 	github.com/charmbracelet/x/ansi v0.4.5 // indirect
+<<<<<<< HEAD
```
Comment (MartinezAvellan) [inline go.mod]: Pls, remove this merge comments. ❤️

File: go.mod [hunk 5]
```diff
@@ -63,7 +70,12 @@ require (
 	github.com/go-openapi/swag v0.23.0 // indirect
 	github.com/golang-jwt/jwt/v4 v4.5.1 // indirect
 	github.com/google/go-cmp v0.6.0 // indirect
+<<<<<<< HEAD
+	github.com/grpc-ecosystem/grpc-gateway/v2 v2.22.0 // indirect
```
Comment (MartinezAvellan) [inline go.mod]: Pls, remove this merge comments. ❤️

File: go.sum [hunk 6]
```diff
@@ -9,10 +9,15 @@ github.com/Masterminds/squirrel v1.5.4 h1:uUcX/aBc8O7Fg9kaISIUsHXdKuqehiXAMQTYX8
 github.com/Masterminds/squirrel v1.5.4/go.mod h1:NNaOrjSoIDfDA40n7sr2tPNZRfjzjA400rg+riTZj10=
 github.com/Microsoft/go-winio v0.6.1 h1:9/kr64B9VUZrLm5YYwbGtUJnMgqWVOdUAXu6Migciow=
 github.com/Microsoft/go-winio v0.6.1/go.mod h1:LRdKpFKfdobln8UmuiYcKPot9D2v6svN5+sAH+4kjUM=
+<<<<<<< HEAD
```
Comment (MartinezAvellan) [inline go.sum]: Pls, remove this merge comments. ❤️

File: go.sum [hunk 7]
```diff
@@ -41,11 +46,21 @@ github.com/charmbracelet/lipgloss v1.0.0 h1:O7VkGDvqEdGi93X+DeqsQ7PKHDgtQfF8j8/O
 github.com/charmbracelet/lipgloss v1.0.0/go.mod h1:U5fy9Z+C38obMs+T+tJqst9VGzlOYGj4ri9reL3qUlo=
 github.com/charmbracelet/x/ansi v0.4.5 h1:LqK4vwBNaXw2AyGIICa5/29Sbdq58GbGdFngSexTdRM=
 github.com/charmbracelet/x/ansi v0.4.5/go.mod h1:dk73KoMTT5AX5BsX0KrqhsTqAnhZZoCBjs7dGWp4Ktw=
+<<<<<<< HEAD
```
Comment (MartinezAvellan) [inline go.sum]: Pls, remove this merge comments. ❤️

**Review by MartinezAvellan — CHANGES_REQUESTED** [review]

**Review by MartinezAvellan — APPROVED** [review]
lgtm.

**Review by MartinezAvellan — APPROVED** [review]

**Review by ClaraTersi — APPROVED** [review]

File: components/mdz/test/integration/testutils.go (hunk 3 above)
Comment (qnen) [reply components/mdz/test/integration/testutils.go:18]: There is some reason to keep this comments? As the conversation was marked as resolved.

**Pull request merged**